base_layer/core/src/types.rs

```diff
@@ -79,7 +79,7 @@ pub const MAX_RANGE_PROOF_RANGE: usize = 64; // 2^64
 /// Current version of the blockchain
 pub const BLOCKCHAIN_VERSION: u16 = 0;
 /// The min required lock height before coinbase utxos are spendable
-pub const COINBASE_LOCK_HEIGHT: u64 = 1440;
+pub const COINBASE_LOCK_HEIGHT: u64 = 1;
```

Comment:
We can't permit changing of consensus variables here. 
If you want this for testing add a `#cfg[test]` section